10-JavaScript/07-jquery-events.md

```diff
@@ -68,10 +69,78 @@ Hints:
 - Detecting keys like space and enter is a little different than letter keys. Look it up!
 - `console.log()` is your friend.
 
+## Advanced Event Handling
+### Binding to Multiple Elements
+In the above examples, we only used ID selectors for jQuery, which means our event was only attached to one object. However, a CSS selector can select multiple elements. [Take this CodePen](http://codepen.io/droberts-ada/pen/jVygpR?editors=1010) for example. When you click on any of the list items, the click handler activates.
+
+Instead of alerting with a generic message, let's tell the user which list item they clicked on. In order to do so, we'll need to figure out which element was clicked. Fortunately, jQuery lets us know which element we're in via `this`.
+
+It's important to be aware that `this` is the raw JavaScript DOM object for the clicked element. If we want access to our nice `jQuery` functionality, just pass it into the `$` function:
+
+```javascript
+$(this)
+```
+
+Armed with `this` knowledge, let's get back to our task: telling the user what element they clicked on. Change the JavaScript in the CodePen to the following:
+
+```javascript
+$(document).ready(function() {
+  $('li').click(function(event) {
+    alert('Got a click on an <li> containing "' + $(this).html() + '"');
+  });
+});
+```
+
+### Event Delegation
```

Comment:
This conversation we might be able to move out since it won't be as relevant for the keyboard assignment (though overall important and they will definitely encounter it at some point)